### PR TITLE
use architect orb

### PR DESCRIPTION
**Pull request by xh3b4sd**
## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,72 +1,56 @@
...
-        ./architect version
-    - run: ./architect build
-    - store_test_results:
-        path: /tmp/results
-    - persist_to_workspace:
-        root: .
-        paths:
-        - ./operatorkit
-        - ./architect
+  architect: giantswarm/architect@0.8.0
```
Comment (xh3b4sd) [inline .circleci/config.yml]:
```suggestion
  architect: giantswarm/architect@0.8.1
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,72 +1,56 @@
...
-        ./architect version
-    - run: ./architect build
-    - store_test_results:
-        path: /tmp/results
-    - persist_to_workspace:
-        root: .
-        paths:
-        - ./operatorkit
-        - ./architect
+  architect: giantswarm/architect@0.8.1
```
Comment (xh3b4sd) [inline .circleci/config.yml]:
```suggestion
  architect: giantswarm/architect@0.8.3
```

File: controller/collector/set.go [hunk 3]
```diff
@@ -26,11 +26,7 @@ func NewSet(config SetConfig) (*Set, error) {
 
 	var timestampCollector *Timestamp
 	{
-		c := TimestampConfig{
-			Logger:               config.Logger,
-			K8sClient:            config.K8sClient,
-			NewRuntimeObjectFunc: config.NewRuntimeObjectFunc,
-		}
+		c := TimestampConfig(config)
```
Comment (kopiczko) [inline controller/collector/set.go]: I didn't even know that's possible.

File: .circleci/config.yml [hunk 4]
```diff
@@ -1,72 +1,56 @@
...
-        ./architect version
-    - run: ./architect build
-    - store_test_results:
-        path: /tmp/results
-    - persist_to_workspace:
-        root: .
-        paths:
-        - ./operatorkit
-        - ./architect
+  architect: giantswarm/architect@dev:drop-unparam
```
Comment (xh3b4sd) [inline .circleci/config.yml]:
```suggestion
  architect: giantswarm/architect@0.8.4
```

**Pull request merged**